terraform/gcp/envs/prod.tfvars

```diff
@@ -8,7 +8,8 @@ admin_url = "juniper-cmi.org"
 environment = "prod"
 # note: automatically creates DNS records for these portals under the admin domain
 
-portals = ["demo", "atcp", "ourhealth", "hearthive", "rgp", "cmi", "trccproject", "gvasc", "pedihccproject"]
+# TODO: remove pedihcc next release
+portals = ["demo", "atcp", "ourhealth", "hearthive", "rgp", "cmi", "trccproject", "gvasc", "pedihccproject", "pedihcc"]
```

Comment:
I used the wrong portal name `pedihcc` on the admin cert so it failed to generate because the DNS would not resolve